File: week_10/day_2/bear.js
var bear ={
  type: "big",
  name: "Bob",
  belly: [],
  eat: function(food) {
    this.belly.push(food)
  }
}

bear.eat("chips cheese and gravy");
bear.eat("lasagne");
bear.eat("fish and chips");
console.log(bear);

File: week6/day_1/multiple_classes_start_point/Bear.java
public class Bear{
  private String name;
  private Salmon[] belly;

  public Bear(String name){
    this.name = name;
    this.belly = new Salmon[5];
  }
  public String getName(){
    return this.name;
  }

  public int foodCount(){
    int counter = 0;
    for (Salmon food : belly){
      if (food != null){
        counter++;
      }
    }
    return counter;
  }

  public void eat(Salmon salmon){
    if ( bellyFull() ){
      return;
    }
    int foods = foodCount();
    belly[foods] = salmon;
  }

  public boolean bellyFull(){
    return foodCount() == belly.length;
  }

  public void sleep(){
    for (int i = 0; i < 5; i++ ){
      belly[i] = null;
    }
  }





}
